### PR TITLE
Portfolio - mobile menu: module 1

### PR DESCRIPTION
**Pull request by yusufyusie**
### Dom implementation on Html elements milestone:- module 1
**This dom implementation includes the following tasks:-**

- Added querySelector to mobile menu selectors of CSS to my dom file of the portfolio
- Added querySelector to the links selector of CSS to the dom file of the portfolio
- Added querySelector to close button of CSS selector in my portfolio

### CODE REVIEW

**Comment by nahnahdev** [comment]
## Project Status: INVALIDATED :red_circle: 
Hi @yusufyusie , 

While you made a great effort in this project, I can't go ahead and review your code.


## Invalid Code Review Request

- This is because you have submitted a project, that is missing the `.eslintrc.json file` which is in charge of the `Eslint` checks or JAVASCRIPT LINTER CHECKS so kindly ensure that you have properly installed the [HTML, CSS, AND JAVASCRIPT LINTERS](https://github.com/microverseinc/linters-config/tree/master/html-css-js) in your project :100: 
- Your Current checks 
image

- What you should have
image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @NahnahAJ in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by mukhammadrizooff — COMMENTED** [review]
Hi @yusufyusie, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that linters don't pass, kindly fix it 🙏🏼 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by yusufyusie** [comment]
https://microverse-students.slack.com/archives/C47D4N35M/p1686654751539179

File: assets/js/mobilemenu.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+linkContact.addEventListener('click', () => {
+  linkContact.classList.add('hidden');
+  linkContact.classList.remove('hidden');
+});
+
+linkPortfolio.addEventListener('click', () => {
+  headerNavMobile.classList.add('hidden');
+  menuBtn.classList.remove('hidden');
+});
```
Comment (AbelG101) [inline assets/js/mobilemenu.js]: - [ ] This is really close! but as part of the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md) please make sure the mobile menu closes when the user clicks (or taps) any of the mobile menu options.

File: .eslintrc.json
Comment (AbelG101) [inline .eslintrc.json:1]: - [ ] I couldn't file your styles file that's why I'm making this comment on this file but I noticed some discrepancies between your design of the mobile menu and the Figma design as you can see from the Figma design the menu doesn't take up the entire height and there also seems to be a rounded radius at the bottom left and right of the menu.
- Figma design:
  image
- Your design:
  image

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @yusufyusie 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Mobile menu opens & closes as expected ✔
-  Well-structured files ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mckent05 — APPROVED** [review]
Hi @yusufyusie, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 
- Good job creating your mobile menu accordingly 👍 👍 

## Optional suggestions
- Please note that your branch name can be more descriptive it could be the name of a feature or an action, in this instance `mobile_menu`. Please take note of this in future microverse projects 👍 👍 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**